app/components/first_draft_collections/create/settings_component.html.erb

```diff
@@ -97,14 +97,14 @@
     </div>
   </section>
 
-  <section id="review">
+  <section id="review" data-controller="review-workflow">
     <header>Review Workflow</header>
 
     <div class="mb-3 row">
       <%= tag.span t('hints.collection.review_enabled'), class: 'col-sm-10 col-xl-8 col-form-label' %>
       <div class="col-sm-3">
         <div class="form-check form-switch">
-          <%= form.check_box :review_enabled, { class: 'form-check-input' }, 'true', 'false' %>
+          <%= form.check_box :review_enabled, { class: 'form-check-input', data: {action: 'review-workflow#toggle', review_workflow_target: 'enabled'} }, 'true', 'false' %>
```

Comment:
please add spaces inside of braces.

Reply:
I just assumed that rubocop fixes _everything_.